src/index.html

```diff
@@ -1,17 +1,169 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+        onsubmit="onSubmit()"
+        action=""
+        method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label for="Surname">Surname:</label>
+          <input
+            type="text"
+            id="Surname"
+            minlength="5"
+            maxlength="12"
+            autocomplete="off">
```

Comment:
checklist point 3 (all similar places)
```suggestion
          <input
            type="text"
            id="Surname"
            minlength="5"
            maxlength="12"
            autocomplete="off"
          >
```

Reply:
resolved